trulens_eval/trulens_eval/feedback/v2/feedback.py

```diff
@@ -594,16 +693,16 @@ class FeedbackWithExplanation(WithPrompt):
             # TODO: things related to extracting score and reasons
 
             def extract_cot_explanation_of_response(
-                self, response: str, normalize=10
+                self, response: str, normalize=3
             ):
                 if "Supporting Evidence" in response:
                     score = 0
                     for line in response.split("\n"):
                         if "Score" in line:
-                            score = re_0_10_rating(line) / normalize
+                            score = re_configured_rating(line) / normalize
```

Comment:
do we need to pass normalize as our max rating here?

Reply:
updated - thanks